src/QMCApp/QMCAppBase.cpp

```diff
@@ -38,6 +38,7 @@ bool QMCAppBase::pushDocument(const std::string& infile)
   else
   {
     app_error() << "File " << infile << " is invalid" << std::endl;
+    APP_ABORT("QMCAppBase::pushDocument");
```

Comment:
returning false is self-consistent. abort is handled outside. I could not agree with this change.